lib/ansible/plugins/loader.py

```diff
@@ -13,7 +13,7 @@
 import sys
 import warnings
 
-from collections import defaultdict, namedtuple
+from collections import defaultdict, namedtuple, OrderedDict
```

Comment:
Is there a specific reason `OrderedDict` is needed?  Insertion order of dicts has been official since py3.7.